sdk/daml-lf/archive/src/main/protobuf/com/digitalasset/daml/lf/archive/daml_lf1.proto

```diff
@@ -1480,9 +1433,6 @@ message Update {
     CreateInterface create_interface = 12; // *Available in versions >= 1.15*
     ExerciseInterface exercise_interface = 13; // *Available in versions >= 1.15*
     FetchInterface fetch_interface = 14; // *Available in versions >= 1.15*
-    DynamicExercise dynamic_exercise = 15; // *Available in versions >= 1.dev*
```

Comment:
These dynamic exercise deletes, presumably these are left overs from the previous PR (and not a rebase issue)?

Reply:
it is a mistake, I should not touch LF 1.x

Reply:
change reverted